game-core/src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -938,9 +937,6 @@ private void parseEditableProperty(
       case "boolean":
         editableProperty = new BooleanProperty(name, null, Boolean.parseBoolean(defaultValue));
         break;
-      case "file":
-        editableProperty = new FileProperty(name, null, defaultValue);
-        break;
```

Comment:
This is the only sort-of breaking change in this PR (and the resulting change in game.dtd).
None of the existing maps in `triplea-maps` seem to use this feature at all.
I highly doubt this really works, because the file is internally represented as `java.io.File` which can't represent files within a zip file at all, so all this property could do is reference files somewhere else on a users PC which likely isn't what we intend to do with such a property. I'm not 100% sure on this one, but AFAIK this property would need to be used by the engine explicitly (can't be referenced by the map itself somewhere), so all that really breaks is a feature that was no-op before, right?

I'd like to hear your opinion on this @ron-murhammer

Reply:
Also I have a batch of other "sort-of" breaking changes that I will submit after this PR to avoid merge conflicts

Reply:
@RoiEXLab This is fine to remove. I'm fairly sure that the FileProperty was only used by one of the old puzzle games that we removed support for already. It was used to load a background image or something like that.

Reply:
Ah thanks for the clarification.
That makes sense 👍